python/src/scipp/neutron/diffraction/load.py

```diff
@@ -0,0 +1,82 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# Copyright (c) 2019 Scipp contributors (https://github.com/scipp)
+# @author Simon Heybrock, Neil Vaytet
+
+import scipp as sc
+
+
+def loadCal(Filename="", InstrumentName="",
```

Comment:
1. I would prefer just calling this `load`:
   - It actually loads more than calibration (I think the Mantid alg is mis-named).
   - Otherwise, the user needs to memorize a new function name. Admittedly, they still need to memorize  that they have to use the `load` algorithm from the `diffraction` module, but I cannot think of a better solution now. Can you?

2. I think we should avoid polluting the scipp API with names from Mantid. We should either forward keyword args unchanged (if required), or provide native variants. In particular, `scipp.neutron.load` uses a `filename` arg, should this be the same here (internally this forwards to `Filename`, but this should not be exposed to the user)? I am not sure what to do with the various options for specifying the instrument. Having this in an algorithm that just loads a file does not feel like a good solution? Does it actually work if you do not specify any of those? I had a look at `LoadDiffCal`, and it appears to fall back to loading the instrument from the file.

Reply:
1. I think load should be reserved for a future wrapper function that handle more cases than just the calibration file like it is done in compat. I actually think the calibration name is fitting since no experiment data is loaded, but only data related to the instrument / source / detectors.
2. This is a tough balance, as finding a new unique set of names is confusing if all the functionality of the Mantid algorithm is desired. If we cut features from the Mantid algorithms, it becomes easier as there are fewer arguments to pass. One option would be to use standard positional arguments for the Filename and InstrumentName as they have to be specified regardless, in this way avoiding the renaming altogether for those variables? The rest could be handled by dict as you propose in another comment.

Reply:
Regarding 1., I propose naming it `load_calibration` then, since we adopt the Python-like convention, not the CamelCase Mantid uses.

Regarding 2., maybe we can argue from the point of how it *should* work: We would want the instrument to be part of the file (just like for our raw files), so specifying `InstrumentName` is very non-standard / old-fashioned-hack. That is, only make `filename` a named argument, everything else goes into the dict of "special" inputs for Mantid.

Reply:
The load_calibration name seems good and I also like keeping the filename the only required parameter with the remaining in the dict. For now the InstrumentName will however be a required parameter in this dict, but this hopefully will not persist.